cmp/7e7d33d5-7101-47f6-a358-1126c7876aa4/component.lp

```diff
@@ -0,0 +1,94 @@
+(librepcb_component 7e7d33d5-7101-47f6-a358-1126c7876aa4
+ (name "Buffer With Inverted Enable Quad")
+ (description "Quad buffer/line driver; 3-state")
+ (keywords "74HCT125, 74HC125, 74XX125, buffer, driver, line-driver, quad-buffer, quad buffer")
+ (author "Shibin Das")
+ (version "0.1")
+ (created 2019-04-07T20:07:35Z)
+ (deprecated false)
+ (category 56f01b1d-0723-4c33-88f7-9968cf96b50f)
+ (category a4c3b94b-5321-4adb-8a77-d3b53f178cee)
+ (schematic_only false)
+ (default_value "74XX125")
+ (prefix "IC")
+ (signal a10f32b8-dec9-4894-92ff-ba1ae5e239c0 (name "2") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal a22f2a47-3a46-4fb0-a44e-194d655393a4 (name "5") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 0246a736-2dd3-40e2-995a-f4e48f98739e (name "13") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal eeffe136-27f1-4232-b94a-b2f01fb583da (name "3") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 59ba36d7-b380-47b0-915a-f5e74dccdd43 (name "8") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal c8be1507-9d6c-412d-a27a-972321f2dbcd (name "11") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal b776bbec-5b3d-49fc-8df2-0dd4067a5182 (name "6") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal db72d9d0-9435-4f2c-b1eb-a5d3b6824856 (name "7") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 2d6148b8-c199-4c55-b7e1-ae9c8daf7d47 (name "10") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal e1488761-b277-4435-9b4a-8accbe814f7e (name "4") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 80d95b99-dbac-40af-9a6c-24a722f2185c (name "12") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 47891ed2-4360-4240-b37b-20b87fb9c91b (name "14") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 9e46c97f-f553-45a6-8ff4-d59d6861b93f (name "1") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 7141a3aa-4f90-41f8-b08e-bf2087d4a136 (name "9") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (variant 53bdfbd3-babf-42c9-9b35-511e342f1284 (norm "")
+  (name "default")
+  (description "")
+  (gate 2df2652b-f5c0-4e6e-9b9f-2073114d8734
+   (symbol ca3e6c8e-5035-4685-a043-0f73c891df55)
+   (position 0.0 0.0) (rotation 0.0) (required true) (suffix "1")
```

Comment:
You should specify a different position for each symbol in the component, otherwise all symbols overlap in the component preview (it's only needed for the preview, it has no other functionality). See the OpAmp Quad as an example.